### ISSUE TITLE
Task list for the competition

### ISSUE THREAD

**Issue by theahmedomar** [Qetar/IDAO-2022#1]
- [ ] Find optimal hyperparameters 
    - [ ]  learning rate
    - [ ] epochs
    - [ ] nfeat_bond
    - [ ] gaussian width
    - [ ] embedding dimensions
    - [ ] hidden layers size
    - [ ] dropout vs no dropout
- [ ] Transfer learning
- [ ] Train with valid structures only [like here](https://github.com/materialsvirtuallab/megnet#training-a-new-megnetmodel-from-structures)
- [ ] Increase dataset size with similar crystals [from here](https://figshare.com/articles/dataset/Graphs_of_materials_project/7451351)
